source/class/qx/ui/form/MenuButton.js

```diff
@@ -142,6 +156,9 @@ qx.Class.define("qx.ui.form.MenuButton",
 
       if (menu)
       {
+        if (this.isFocusable() && !qx.ui.core.FocusHandler.getInstance().isFocused(this)) {
```

Comment:
Also a change of behavior, but looks like it actually fixes a bug.

Reply:
Agree, but needed. If open is called the Button didn't get focused